build_aarch64_wheel.py

```diff
@@ -161,27 +161,35 @@ def start_build(key_name, *,
     print('Checking out PyTorch repo')
     run_ssh(addr, f"git clone --recurse-submodules -b {branch} https://github.com/pytorch/pytorch {git_clone_flags}")
     print('Building PyTorch wheel')
+    build_vars=""
     if branch == 'nightly':
         build_date = check_output(addr, "cd pytorch ; git log --pretty=format:%s -1").strip().split()[0].replace("-", "")
         version = check_output(addr, "cat pytorch/version.txt").strip()[:-2]
-        run_ssh(addr, f"cd pytorch ; BUILD_TEST=0 PYTORCH_BUILD_VERSION={version}.dev{build_date} PYTORCH_BUILD_NUMBER=1 python3 setup.py bdist_wheel")
-    else:
-        run_ssh(addr, "cd pytorch ; python3 setup.py bdist_wheel")
+        build_vars += f"BUILD_TEST=0 PYTORCH_BUILD_VERSION={version}.dev{build_date} PYTORCH_BUILD_NUMBER=1"
+    if branch.startswith("v1."):
+        build_vars += f"BUILD_TEST=0 PYTORCH_BUILD_VERSION={branch[1:branch.find('-')]} PYTORCH_BUILD_NUMBER=1"
+    run_ssh(addr, f"cd pytorch ; {build_vars} python3 setup.py bdist_wheel")
     pytorch_wheel_name = list_dir(addr, "pytorch/dist")[0]
     embed_libgomp(addr, use_conda, os.path.join('pytorch', 'dist', pytorch_wheel_name))
     print('Copying the wheel')
     subprocess.check_call(["scp", "-i", keyfile_path, f"ubuntu@{addr}:pytorch/dist/*.whl", "."])
 
     print('Checking out TorchVision repo')
-    run_ssh(addr, f"git clone https://github.com/pytorch/vision {git_clone_flags}")
+    if branch.startswith("v1.7.1"):
+        run_ssh(addr, f"git clone https://github.com/pytorch/vision -b v0.8.2-rc2 {git_clone_flags}")
```

Comment:
interesting :-)

Reply:
I can't think of a better way of doing that, perhaps we can have a version correspondence dictionary later